src/Service/AbstractNavigationFactory.php

```diff
@@ -151,7 +150,7 @@ protected function injectComponents(
             $hasUri = isset($page['uri']);
             $hasMvc = isset($page['action']) || isset($page['controller']) || isset($page['route']);
             if ($hasMvc) {
-                if (! isset($page['routeMatch']) && $routeMatch) {
+                if (! isset($page['routeMatch'])) {
```

Comment:
Not sure this conditional should be dropped: `$routeMatch` **can** be `null` here? :thinking:

Reply:
There's a call just before to `assertRouteMatch` or similar - psalm rightly picked up the redundancy

Reply:
Sorry, nope - you're right - might be null - need to fix!!